pythonFiles/vscode_datascience_helpers/getVariableInfo/vscodeGetVariableInfo.py

```diff
@@ -13,8 +15,42 @@ def truncateString(string):
         else:
             return string
 
+    DisplayOptions = _VSCODE_namedtuple("DisplayOptions", ["width", "max_columns"])
+
+    def set_pandas_display_options(display_options=None):
+        if _VSCODE_importlib_util.find_spec("pandas") is not None:
+            try:
+                import pandas as _VSCODE_PD
```

Comment:
YOu will want to delete this again at the end of the try.. in finally
`del _VSCODE_PD`

Reply:
good catch

Reply:
Not sure these are actually necessary - the function scope seems to take care of this:
![image](https://github.com/microsoft/vscode-jupyter/assets/2019016/9f69fc99-b5ba-4a0c-aa9d-4e4fbe925c1e)